dependencies/pom.xml

```diff
@@ -16,6 +20,22 @@
     <name>Google LLC.</name>
     <url>https://www.google.com</url>
   </organization>
+  <inceptionYear>2018</inceptionYear>
+  <developers>
+    <developer>
```

Comment:
@suztomo  also for these poms?

Reply:
PII should only be included at explicit request.